package.json

```diff
@@ -51,7 +51,7 @@
     "publish:public": "yarn release && yarn build && git push origin HEAD && git push --tags && yarn publish",
     "publish:public:dryrun": "yarn release --dry-run",
     "release": "standard-version",
-    "start": "esbuild src/index.js --bundle --global-name=ole --loader:.svg=dataurl --minify --outfile=index.js --serve --servedir=. --sourcemap --watch",
+    "start": "esbuild src/index.js --bundle --global-name=ole --loader:.svg=dataurl --minify --outfile=index.js --serve --servedir=. --sourcemap --watch=forever",
```

Comment:
I start often the server my PC will crash :D . Could we do a start:cy target:

"start:cy": yarn start --watch=forever

or maybe using cy:test directly:

"cy:test": "start-server-and-test \"yarn start --watch=forever\" http://127.0.0.1:8000 cy:run",

Reply:
esbuild will use the last --watch property . Id o that with --outfile property in sev

Reply:
I've tested running `yarn start` with `--watch=forever`, once I stop the Yarn task all related Node.js process got killed. Can you check if they still run on your machine after stopping the Yarn task?

Reply:
yes

Reply:
@friedjoff  you re right. it s good for me